shared/templates/csv/sshd_lineinfile.csv

```diff
@@ -0,0 +1,2 @@
+# <rule_id, parameter, value[, missing_parameter_pass]> (missing_parameter_pass: If set, the OVAL check will also pass if the parameter is not present in the configuration file. Values can be true or false)
+# sshd_enable_strictmodes,StrictModes,yes,true
```

Comment:
Was it intention to comment this? I see you didn't delete oval, bash and ansible for the rule, but just to be sure.

Reply:
It was supposed to serve as example doc. I added doc explaining that it's an example of usage.